sass/components/_offender-basic-details.scss

```diff
@@ -0,0 +1,38 @@
+.bold {
+  font-weight: bold
+}
+
+.basic-details {
+  border-top: 1px solid #b1b4b6;
+  border-bottom: 1px solid #b1b4b6;
+  padding-top: 15px;
+  padding-bottom: 15px;
```

Comment:
you shouldn't really need to add so much custom styling if using gds - there are custom built classes that you can use. https://design-system.service.gov.uk/styles/spacing/

Reply:
similar with typography too

Reply:
Be worth looking at the other pages Jon.

Reply:
This is matching Pathfinder, so for consistency wanted to keep it as close to that as possible.  I'll take a look to reduce this though.